rdfloader/parser2v2/license_utils.go

```diff
@@ -80,12 +80,12 @@ func mapLicensesToStrings(licences []AnyLicenseInfo) []string {
 
 /****** Type Functions ******/
 
-// TODO: should probably add brackets while linearizing a nested license.
+// ToLicenseString TODO: should probably add brackets while linearizing a nested license.
```

Comment:
avoid adding Function names before the function unless the following statement captures the intent of the function.